lib/commands/recordscreen.js

```diff
@@ -210,6 +215,37 @@ commands.startRecordingScreen = async function (options = {}) {
       `The value of '${timeLimit}' has been passed instead.`);
   }
 
+  let {
+    mjpegServerScreenshotQuality,
+    mjpegServerFramerate,
+  } = await this.proxyCommand('/appium/settings', 'GET');
+  if (videoQuality) {
+    const quality = _.isInteger(quality) ? quality : QUALITY_MAPPING[_.toLower(videoQuality)];
+    if (!quality) {
+      throw new Error(`videoQuality value should be one of ${JSON.stringify(_.keys(QUALITY_MAPPING))} or a number in range 1..100. ` +
+        `'${videoQuality}' is given instead`);
+    }
+    mjpegServerScreenshotQuality = mjpegServerScreenshotQuality !== quality ? quality : undefined;
+  } else {
+    mjpegServerScreenshotQuality = undefined;
+  }
+  if (videoFps) {
+    const fps = parseInt(videoFps, 10);
+    if (isNaN(fps)) {
+      throw new Error(`videoFps value should be a valid number in range 1..60. ` +
+        `'${videoFps}' is given instead`);
```

Comment:
Shouldn't we verify that `1 <= videoFps <= 60`

Reply:
no, this is done automatically by WDA